### PR TITLE
Update applens tenant

### PR DESCRIPTION
**Pull request by nwnt**
### Which issue this PR addresses:

Urgent update the AppLens tenant.

### What this PR does / why we need it:

AppLens will be enforcing AAD Auth in AME on March 13th.

### Test plan for issue:
N/A -- Config update.

### Is there any documentation that needs to be updated for this PR?
N/A

### CODE REVIEW

**Review by hlipsig — DISMISSED** [review]

**Review by s-amann — APPROVED** [review]

**Review by weinong — APPROVED** [review]
LGTM

File: pkg/util/azureclient/environments.go [hunk 1]
```diff
@@ -50,8 +50,8 @@ var (
 		GenevaMonitoringEndpoint: "https://gcs.prod.monitoring.core.windows.net/",
 		AppSuffix:                "aro.azure.com",
 		AppLensEndpoint:          "https://diag-runtimehost-prod.trafficmanager.net/api/invoke",
-		AppLensScope:             "b9a1efcd-32ee-4330-834c-c04eb00f4b33",
-		AppLensTenantID:          "72f988bf-86f1-41af-91ab-2d7cd011db47",
+		AppLensScope:             "0d7b6142-46a3-426a-ad6d-eed97c2a48ee",
+		AppLensTenantID:          "33e01921-4d64-4f8c-a055-5bdaffd5e33d",
```
Comment (SudoBrendan) [inline pkg/util/azureclient/environments.go:54]: Just checking - we don't need FairFax updated as well bellow?

**Review by cadenmarchese — APPROVED** [review]

**Review by SudoBrendan — APPROVED** [review]
LGTM as long as FF is out of scope.

**Review by hlipsig — APPROVED** [review]

File: pkg/util/azureclient/environments.go (hunk 1 above)
Comment (nwnt) [reply pkg/util/azureclient/environments.go:54]: nope, the scope of the changes is for apps authenticated with msft tenant.  They have to authenticate with AME starting from March 13.

**Pull request merged**